README.md

```diff
@@ -21,6 +21,19 @@ To bind the java agent to a specific IP change the port number to `host:port`.
 
 See `./run_sample_httpserver.sh` for a sample script that runs the httpserver against itself.
 
+### Wildfly
```

Comment:
I can and will try but they aren't exactly responsive. I'll see what they say on the mailing list and try to add a comment to the JMX page of the documentation. With respect, I'd appreciate you allowing this to be merged given its only a small amount of documentation.